addons/l10n_es/controllers/portal.py

```diff
@@ -1,19 +1,20 @@
-from odoo.addons.website_sale.controllers.main import WebsiteSale
 from odoo.http import request
 
+from odoo.addons.portal.controllers.portal import CustomerPortal
```

Comment:
You could maybe inherit from the account `PortalAccount` controller instead to have an inheritance tree following the modules dependencies ? :thinking: